lib/screens/home/index.dart

```diff
@@ -3,18 +3,26 @@ import 'package:flutter/material.dart';
 import 'package:learning_compass_exp/screens/home/home.dart';
 import 'package:learning_compass_exp/screens/home/widgets/incrementer_button.dart';
 
+import 'widgets/drawer/drawer.dart';
+
 
 // This Widget defines the root of the Home-screen
 class HomeScreen extends StatelessWidget {
+
+  final MDrawerWidget drawer = new MDrawerWidget() ;
```

Comment:
I thought it will be nice to distinguish my widgets from the provided widgets. I will change it anyway.  Maybe we have to discuss the naming convention we will be using.

Reply:
Oh, maybe you should name the drawer according to what screen it's on, as in 'HomeDrawer' (or perhaps MenuDrawer). I don't think this particular drawer will be used elsewhere.

Reply:
I agree with Olli, I think we should prefer naming widgets according to their purpose. In this instance, instead of naming the drawer with M as the initial character, naming it 'HomeDrawer' or 'MenuDrawer' (or something similar) as suggested by Olli would be a more descriptive and therefore also easier and faster to understand when reading the code.